apps/molgenis-components/src/components/forms/InputLong.vue

```diff
@@ -0,0 +1,117 @@
+<template>
+  <span>
+    <FormGroup
+      :id="id"
+      :label="label"
+      :description="description"
+      :errorMessage="longError"
+      v-on="$listeners"
+    >
+      <input
+        :value="value"
+        :class="{ 'form-control': true, 'is-invalid': errorMessage }"
+        :aria-describedby="id + 'Help'"
+        :placeholder="placeholder"
+        :readonly="readonly"
+        @keypress="handleKeyValidity($event)"
+        @input="inputHandler($event)"
+      />
+    </FormGroup>
+  </span>
+</template>
+
+<script>
+import FormGroup from "./FormGroup.vue";
+import BaseInput from "./BaseInput.vue";
+import constants from "../constants";
+import { isNumericKey } from "./utils/InputUtils";
+
+const { CODE_MINUS, MIN_LONG, MAX_LONG } = constants;
+
+export default {
+  extends: BaseInput,
+  components: {
+    FormGroup,
+  },
+  props: {
+    errorMessage: { type: String, default: null },
+    readonly: {
+      type: Boolean,
+      required: false,
+      default: () => undefined,
+    },
+  },
+  computed: {
+    longError() {
```

Comment:
same as decimalInput, I think errorMessage is a better name for this :) (though I don't mind this one as much)